core/reporter-impl/pom.xml

```diff
@@ -78,6 +78,12 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>21.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This is because of the line: https://github.com/arquillian/arquillian-reporter/pull/15/files#diff-1f43dc9359d1729a792ef0c4c821e8c5R74 right?
I think that it doesn't bring any additional value, so we can use just normal HashMap and to not add an additional dependency.
Correct me If I'm wrong